server/lib/src/entry.rs

```diff
@@ -358,6 +358,8 @@ impl Entry<EntryInit, EntryNew> {
     pub(crate) fn unsafe_from_entry_str(es: &str) -> Self {
         // Just use log directly here, it's testing
         // str -> proto entry
+
```

Comment:
I swear this function is one of my deepest regrets.

Reply:
we'll excise it one day!